.github/workflows/performance.yml

```diff
@@ -0,0 +1,152 @@
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements. See the NOTICE file distributed with
+#  this work for additional information regarding copyright ownership.
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License. You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+name: Performance
+
+on:
+  issue_comment:
+  workflow_dispatch:
+
+jobs:
+  pr_comment:
+    name: Benchmark by comment
+    if: ${{ (github.event.issue.pull_request && contains(github.event.comment.body, '/perf')) }}
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: xt0rted/pull-request-comment-branch@v1
+        id: comment-branch
+
+      - name: 'Check out repository'
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ steps.comment-branch.outputs.head_ref }}
+          submodules: 'true'
+
+      - name: 'Set up Java'
+        uses: actions/setup-java@v3
+        with:
+          java-version: 19
+          distribution: 'zulu'
+
+      - name: 'Cache Maven packages'
+        uses: actions/cache@v3
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+
+      - name: 'Build kroxy service'
+        run: mvn -B clean verify -Pdist -Dquick
+
+      - name: 'Download kafka'
+        run: wget -q https://downloads.apache.org/kafka/3.4.0/kafka_2.13-3.4.0.tgz; tar xf kafka_2.13-3.4.0.tgz
```

Comment:
Can we extract the kafka version to a variable?

Reply:
done